### PR TITLE
Deleted gainFocus and loseFocus

### PR DESCRIPTION
**Pull request by chessky**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/forgotpassword/DetachedForgotPasswordPage.java [hunk 1]
```diff
@@ -48,10 +44,8 @@ public boolean submitButtonNotClickable() {
 
   @Override
   public void requestLinkForUsername(String username) {
-    gainFocus();
     forgotPasswordPage.requestLinkForUsername(username);
     if(forgotPasswordPage.isConfirmationDisplayed()) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/forgotpassword/DetachedForgotPasswordPage.java]: MAJOR Either remove or fill this block of code. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00108)

File: src/test/java/com/wikia/webdriver/testcases/Anonymization/AnonymizationTests.java [hunk 2]
```diff
@@ -22,6 +22,8 @@
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.ArticleHistoryPage;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.anonymization.SpecialAnonymizationUserPage;
 
+import org.apache.tools.ant.taskdefs.Java;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/Anonymization/AnonymizationTests.java]: MINOR Remove this unused import 'org.apache.tools.ant.taskdefs.Java'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/Anonymization/AnonymizationTests.java [hunk 3]
```diff
@@ -22,6 +22,8 @@
...
+import org.apache.tools.ant.taskdefs.Sleep;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/Anonymization/AnonymizationTests.java]: MINOR Remove this unused import 'org.apache.tools.ant.taskdefs.Sleep'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 8 issues
* ![MAJOR][MAJOR] 3 major
* ![MINOR][MINOR] 5 minor

Watch the comments in this conversation to review them.

#### 5 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AnonymizationTests.java#L1](https://github.com/Wikia/selenium-tests/blob/8a1736afd4f8543a4da5da007ca6dc9635a75457/src/test/java/com/wikia/webdriver/testcases/Anonymization/AnonymizationTests.java#L1): Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00120)
1. ![MAJOR][MAJOR] [AnonymizationTests.java#L109](https://github.com/Wikia/selenium-tests/blob/8a1736afd4f8543a4da5da007ca6dc9635a75457/src/test/java/com/wikia/webdriver/testcases/Anonymization/AnonymizationTests.java#L109): Remove this unused "id" local variable. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)
1. ![MINOR][MINOR] [AnonymizationTests.java#L50](https://github.com/Wikia/selenium-tests/blob/8a1736afd4f8543a4da5da007ca6dc9635a75457/src/test/java/com/wikia/webdriver/testcases/Anonymization/AnonymizationTests.java#L50): Define a constant instead of duplicating this literal "QAanon" 4 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [AnonymizationTests.java#L51](https://github.com/Wikia/selenium-tests/blob/8a1736afd4f8543a4da5da007ca6dc9635a75457/src/test/java/com/wikia/webdriver/testcases/Anonymization/AnonymizationTests.java#L51): Define a constant instead of duplicating this literal "makak" 4 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [AnonymizationTests.java#L106](https://github.com/Wikia/selenium-tests/blob/8a1736afd4f8543a4da5da007ca6dc9635a75457/src/test/java/com/wikia/webdriver/testcases/Anonymization/AnonymizationTests.java#L106): Define a constant instead of duplicating this literal "AnonymizationTest" 6 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by nikodamn** [comment]
Still valid?

**Pull request closed without merging**